libcloud/compute/drivers/outscale.py

```diff
@@ -7948,3 +7949,283 @@ def _to_key_pair(self, key_pair):
             private_key=private_key,
             fingerprint=key_pair["KeypairFingerprint"],
             driver=self)
+
+    def ex_create_master_key(
+        self,
+        description: str = "",
```

Comment:
Same as for the other PR
`description: str = None` and `if description is  not None:`